packages/tangle/tipmanager.go

```diff
@@ -215,17 +215,19 @@ func (t *TipManager) checkApprovers(messageID MessageID) bool {
 }
 
 func (t *TipManager) removeStrongParents(message *Message) {
-	message.ForEachParentByType(StrongParentType, func(parentMessageID MessageID) {
+	message.ForEachParentByType(StrongParentType, func(parentMessageID MessageID) bool {
```

Comment:
Can't comment on the actual line but it seems that `checkApprovers` should only check for strong parents no?